neps/nep-0330.md

```diff
@@ -25,16 +29,23 @@ There is a lot of information that can be held about a contract. Ultimately, we
 
 ## Specification
 
-Successful implementations of this standard will introduce a new  (`ContractMetadata`) struct that will hold all the necessary information to be queried for. This struct will be kept on the contract level.
+Successful implementations of this standard will introduce a new  (`ContractSourceMetadata`) struct that will hold all the necessary information to be queried for. This struct will be kept on the contract level.
 
-The metadata will include two optional fields:
+The metadata will include three optional fields:
 - `version`: a string that references the specific commit hash or version of the code that is currently deployed on-chain. This can be included regardless of whether or not the contract is open-sourced and can also be used for organizational purposes.
 - `link`: a string that references the link to the open-source code. This can be anything such as Github or a CID to somewhere on IPFS.
+- `standards`: a list of objects (see type definition below) that enumerates the NEPs supported by the contract. If this extension is supported, it is advised to also include NEP-330 version 1.1.0 in the list (`{standard: "nep330", version: "1.1.0"}`).
```

Comment:
@esaminu I have added the `standards` field description here as you mentioned before.